examples/xata/README.md

```diff
@@ -0,0 +1,71 @@
+# Xata template
+
+This example showcases how to use Remix with [Xata](https://xata.io) as your data layer.
+
+With this template you get out-of-the-box:
+
+- API Route to connect to your Xata database
+- Type-safe Codegen
+- Accessibility-Ready
+  - Dark/Light mode
+  - Respects `prefers-reduce-motion` for CSS Transitions
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+- [![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/xata)
+- [Check it live](https://remix-with-xata.netlify.app)
+
+## Example
+
+Execute [`create-remix-app`](https://github.com/vercel/next.js/tree/canary/packages/create-remix-app) with [npm](https://docs.npmjs.com/cli/init), [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/), or [pnpm](https://pnpm.io) to bootstrap the example:
+
+```sh
+npx create-remix-app --template xata remix-xata-app
+
+```
+
+<details>
+<summary> Yarn or PNPM</summary>
+
+```sh
+yarn create remix-app --template xata remix-xata-app
+```
+
+```sh
+pnpm create remix-app --template xata remix-xata-app
+```
+
+</details>
+
+### Link Your Xata Workspace and Run Codegen
```

Comment:
It'd be useful to put here that you need to install the xata CLI first.

Reply:
the CLI is in the `devDependencies`, it's not necessary to have it globally